llarp/rpc/rpc_server.cpp

```diff
@@ -128,173 +127,140 @@ namespace llarp::rpc
   {
     if (r->IsServiceNode())
     {
-      reply(CreateJSONError("not supported"));
+      reply(JSONError("not supported"));
       return;
     }
     std::optional<llarp::service::Address> exit;
     std::optional<std::string> lnsExit;
     IPRange range;
     bool map = true;
-    const auto exit_itr = obj.find("exit");
-    if (exit_itr != obj.end())
+
+    if (auto maybe = llarp::json::maybe_get<bool>(obj, "unmap", false))
+    {
+      map = *maybe != true;
```

Comment:
`map = not *maybe`?

Reply:
though that's hideous with the var name "maybe"

Reply:
the meaning is more explicit with `!= true`